backend/sample.env

```
@@ -143,3 +143,6 @@ CELERY_BROKER_URL = "redis://unstract-redis:6379"
 
 # Indexing flag to prevent re-index
 INDEXING_FLAG_TTL=1800
+
+#Publicly whitelisted URL path_prefix
+PUBLIC_PATH_PREFIX=
```

```suggestion
PUBLIC_PATH_PREFIX=public
```
Also @harini-venkataraman why do we need to define an env which is not used in OSS code?

---

Explicitly we don`t use this prefix in OSS. This was added to stop OSS failing altogether. Have handled the exception and removed it.